test/validatorTest.js

```diff
@@ -720,6 +720,24 @@ describe("attestation validation", function() {
 					return assert.isRejected(attResp.validateFlags(), Error, "expected flag was not set: ED");
 				});
 
+				it("throws if UV and UV is set but not UP", function() {
```

Comment:
If UV and UV is set? Is that a typo?

Reply:
I have used the style already present in the test, for example [here](https://github.com/webauthn-open-source/fido2-lib/blob/993521509af7b6e25eb40513a417b7fa31ddacce/test/validatorTest.js#L739): "_throws if UP-or-UV and neither is set_".

Maybe, using the same style, I can write: "_throws if UV and UP is not set_".

Do you have any suggestions to improve the test name?

Reply:
In this case I believe "throws if UV is set but UP is not set" - same for the other one